content/account/app/index.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Overview
+meta_description: " Manage and monitor your applications on the Ably platform using the Ably dashboard. Create new apps, view existing ones, and configure settings from your browser."
+meta_keywords: “Ably dashboard, manage applications, monitor apps, app creation, application settings, Ably platform”
+---
+
+Manage and monitor your applications on the Ably platform using the Ably dashboard. Create new apps, view existing ones, and configure settings from your browser.
+
+Begin by "logging":https://ably.com/login in to Ably through your browser. Once you're logged in, you have access to the Ably "dashboard":https://ably.com/accounts, where you can "create a new app":#create or "view your existing apps.":#view
+
+h2(#create). Create a new app
+
+The following image displays the Ably dashboard when no apps have been created yet. To get started, you can easily *Create new app*:
+
+<a href="@content/screenshots/dash/create-app.png" target="_blank">
+  <img src="@content/screenshots/dash/create-app.png" style="width: 100%" alt="Create apps in the Ably dashboard">
+</a>
+
+h2(#view). View your apps
+
+If you have created apps already, your dashboard will look similar to the one shown in the following image. Each app displayed is a link to its specific dashboard, where you can view and configure settings, as well as monitor usage and statistics:
+
+<a href="@content/screenshots/dash/your-apps.png" target="_blank">
+  <img src="@content/screenshots/dash/your-apps.png" style="width: 100%" alt="Your apps in the Ably dashboard">
+</a>
+
+h2. Dashboard overview
```

Comment:
Maybe 'App overview' if we're calling the whole account the 'dashboard'?

Reply:
I was thinking app dashboard overview originally. I'll go with app overview :)